crates/biome_js_analyze/src/lint/a11y/no_autofocus.rs

```diff
@@ -50,6 +52,16 @@ declare_lint_rule! {
     /// <MyComponent autoFocus={true} />
     ///```
     ///
+    /// ```jsx
+    /// // `autoFocus` prop in element has `popover` attribute is valid
```

Comment:
Nit: generally, when we need to explain a particular case, we tend to add a paragraph before the code, like in these cases

Reply:
Okay, I've perfected it.